lib/src/suggestor_mixins.dart

```diff
@@ -87,28 +77,25 @@ mixin AstVisitingSuggestor<R> on AstVisitor<R> implements Suggestor {
 /// Note that this mixin provides an implementation of [generatePatches] that
 /// should not need to be overridden except for performance optimization reasons
 /// like avoiding analysis on certain files.
-mixin ElementVisitingSuggestor<R> on ElementVisitor<R> implements Suggestor {
-  final _patches = <Patch>{};
+mixin ElementVisitingSuggestor<R> on ElementVisitor<R> {
+  final patches = <Patch>[];
 
   /// The context helper for the file for which patches are currently being
   /// generated.
   FileContext get context => _context;
   FileContext _context;
 
-  @override
-  Stream<Patch> generatePatches(FileContext context) async* {
-    _patches.clear();
-    _context = context;
+  Stream<Patch> suggestor(FileContext context) async* {
```

Comment:
I think this needs to be updated to be `call`

Reply:
👍 good catch